libs/runtime/ebpf_ring_buffer.c

```diff
@@ -146,23 +153,30 @@ ebpf_ring_buffer_output(_Inout_ ebpf_ring_buffer_t* ring, _In_reads_bytes_(lengt
 }
 
 void
-ebpf_ring_buffer_query(_In_ const ebpf_ring_buffer_t* ring, _Out_ size_t* consumer, _Out_ size_t* producer)
+ebpf_ring_buffer_query(_In_ ebpf_ring_buffer_t* ring, _Out_ size_t* consumer, _Out_ size_t* producer)
```

Comment:
as long as SAL does not complain about taking the locks, I think `const` still applies.

Reply:
Yes I had to remove `const` because of SAL warning.